src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/CollectionChangeEventArgs.cs

```diff
@@ -25,6 +25,6 @@ public CollectionChangeEventArgs(CollectionChangeAction action, object element)
         /// <summary>
         /// Gets the instance of the collection with the change.
         /// </summary>
-        public virtual object Element { get; }
+        public virtual object? Element { get; } // TODO-NULLABLE: System.Data.Common doesn't seem to expect this to be null but we do not guard against it
```

Comment:
cc: @roji

Reply:
Yeah, I don't think a null Element is actually usable anywhere in System.Data.Common, but in a general collection which allows nulls it does make sense.

Reply:
Thanks @roji, removing comment in that case